pl_examples/domain_templates/imagenet.py

```diff
@@ -160,91 +157,31 @@ def val_dataloader(self):
-        parser = parent_parser.add_argument_group("ImageNetLightningModel")
-        parser.add_argument(
-            "-a",
-            "--arch",
-            metavar="ARCH",
-            default="resnet18",
-            choices=ImageNetLightningModel.MODEL_NAMES,
-            help=("model architecture: " + " | ".join(ImageNetLightningModel.MODEL_NAMES) + " (default: resnet18)"),
-        )
-        parser.add_argument(
-            "-j", "--workers", default=4, type=int, metavar="N", help="number of data loading workers (default: 4)"
-        )
-        parser.add_argument(
-            "-b",
-            "--batch-size",
-            default=256,
-            type=int,
-            metavar="N",
-            help="mini-batch size (default: 256), this is the total batch size of all GPUs on the current node"
-            " when using Data Parallel or Distributed Data Parallel",
-        )
-        parser.add_argument(
-            "--lr", "--learning-rate", default=0.1, type=float, metavar="LR", help="initial learning rate", dest="lr"
-        )
-        parser.add_argument("--momentum", default=0.9, type=float, metavar="M", help="momentum")
-        parser.add_argument(
-            "--wd",
-            "--weight-decay",
-            default=1e-4,
-            type=float,
-            metavar="W",
-            help="weight decay (default: 1e-4)",
-            dest="weight_decay",
-        )
-        parser.add_argument("--pretrained", dest="pretrained", action="store_true", help="use pre-trained model")
-        return parent_parser
-
-
-def main(args: Namespace) -> None:
-    if args.seed is not None:
-        pl.seed_everything(args.seed)
-
-    if args.accelerator == "ddp":
-        # When using a single GPU per process and per
-        # DistributedDataParallel, we need to divide the batch size
-        # ourselves based on the total number of GPUs we have
-        args.batch_size = int(args.batch_size / max(1, args.gpus))
-        args.workers = int(args.workers / max(1, args.gpus))
-
-    model = ImageNetLightningModel(**vars(args))
-    trainer = pl.Trainer.from_argparse_args(args)
-
-    if args.evaluate:
-        trainer.test(model)
-    else:
-        trainer.fit(model)
-
-
-def run_cli():
-    parent_parser = ArgumentParser(add_help=False)
-    parent_parser = pl.Trainer.add_argparse_args(parent_parser)
-    parent_parser.add_argument("--data-path", metavar="DIR", type=str, help="path to dataset")
-    parent_parser.add_argument(
-        "-e", "--evaluate", dest="evaluate", action="store_true", help="evaluate model on validation set"
-    )
-    parent_parser.add_argument("--seed", type=int, default=42, help="seed for initializing training.")
-    parser = ImageNetLightningModel.add_model_specific_args(parent_parser)
-    parser.set_defaults(profiler="simple", deterministic=True, max_epochs=90)
-    args = parser.parse_args()
-    main(args)
-
 
 if __name__ == "__main__":
-    cli_lightning_logo()
```

Comment:
The logo is removed. Showing it is annoying because most people copy paste this file into their local system and this requires the entire repository as it is defined in the `pl_examples` directory